README.md

```diff
@@ -328,22 +328,22 @@ If you have ever logged into a service by using your Twitter, Google, or Faceboo
 
 OAuth Providers (Facebook, Google, etc), operate through private, unique, access tokens that provide the means of authentication for your service (the "OAuth client") to allow logins.
 
-If you want to use OAuth for your users to login to your service, you will need to register your server as an OAuth Client. This will usually set you up with a `client id`, and `client secret`. Users that login to your service will be relocated to the OAuth Provider where the user can confirm that they do indeed want to "login" (i.e. allow the server they are logging in to, to have access to any required information from the OAuth Provider. )
+If you want to use OAuth for your users to log into your service, you will need to register your server as an OAuth Client. This will usually set you up with a `client id`, and `client secret`. Users that log in to your service will be relocated to the OAuth Provider where the user can confirm that they do indeed want to "log in" (i.e. allow the server they are logging into, to have access to any required information from the OAuth Provider. )
 
 In the case of our friend Beorn...
 
 - Beorn goes to `http://knittingworld.com` to get some nice yarn.
-- Beorn decides to login using his Google account.
-- Beorn is prompted to enter his google account credentials (in the case that he isn't already logged in)
+- Beorn decides to log in using his Google account.
+- Beorn is prompted to enter his Google account credentials (in the case that he isn't already logged in)
 - After entering his credentials, Google (or whatever OAuth provider he uses) will prompt him to check that he wants to sign into `http://knittinggworld.com` with his Google account.
 - After accepting, Beorn is redirected to `http://knittingworld.com`.
-- If `knittingworld` needs access to resource regarding Beorn's information it can make requests to a `resource server` (via the OAuth provider) to access them, provided its access token is valid.
+- If `knittingworld` needs access to a resource regarding Beorn's information, it can make requests to a `resource server` (via the OAuth provider) to access them, provided its access token is valid.
 
 
 From the [OWASP Authentication Cheat Sheet](https://www.owasp.org/index.php/Authentication_Cheat_Sheet#OAuth)
->The recommendation is to use and implement OAuth 1.0a or OAuth 2.0, since the very first version (OAuth1.0) has been found to be vulnerable to session fixation.
+>The recommendation is to use and implement OAuth 1.0a or OAuth 2.0 since the very first version (OAuth1.0) has been found to be vulnerable to session fixation.
```

Comment:
This is technically correct with and without the comma. I prefer it _with_ the comma since that makes the `since the very first version...` phrase read as a parenthetical/aside.